src/compojure/api/meta.clj

```diff
@@ -151,7 +151,7 @@
       "  (ok))")))
 
 (defmethod restructure-param :no-doc [_ v acc]
-  (update-in acc [:swagger] assoc :x-no-doc v))
+  (update-in acc [:info] assoc ::no-doc v))
```

Comment:
should this be non-qualidied? as it's a system value, qualified could be left for user extension?